helm/stacks-blockchain/README.md

```diff
@@ -0,0 +1,136 @@
+
+To install the chart with the release name `my-release` and run the node as a **follower**:
+
+```bash
+$ helm repo add blockstack https://charts.blockstack.xyz
+$ helm install my-release blockstack/stacks-blockchain
+```
+
+To install the chart with the release name `my-release` and run the node as a **miner** using your private key [generated from the instructions on this page](https://docs.blockstack.org/mining):
+
+```bash
+$ helm repo add blockstack https://charts.blockstack.xyz
+$ helm install my-release blockstack/stacks-blockchain --set config.node.miner=true --set config.node.seed="REPLACE-WITH-YOUR-PRIVATE-KEY"
+```
+
+These commands deploy the Stacks Blockchain node on a Kubernetes cluster in the default configuration.
+
+> **Tip**: List all releases using `helm list`
+
+## Uninstalling the Chart
+
+To uninstall/delete the `my-release` deployment:
+
+```bash
+$ helm delete my-release
+```
+
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## Parameters
+
+The following tables lists the configurable parameters of the stacks-blockchain chart and their default values.
+
+| Parameter | Description | Default |
+|-|-|-|
+| nameOverride | String to partially override stacks-blockchain.fullname template (will maintain the release name) | nil |
+| fullnameOverride | String to fully override stacks-blockchain.fullname template | nil |
+| node.image.repository | stacks-blockchain container image repository | blockstack/stacks-blockchain |
+| node.image.pullPolicy | stacks-blockchain container image pull policy | IfNotPresent |
+| node.image.tag | stacks-blockchain container image tag | v23.0.0.10-krypton |
+| node.imagePullSecrets | Docker registry secret names as an array | [] |
+| node.replicaCount | Number of stacks-blockchain nodes to run | 1 |
+| node.rpcPort | Port used for RPC connections | 20443 |
+| node.p2pPort | Port used for P2P connections | 20444 |
+| node.labels | Labels to be added to the node Deployment | {} |
+| node.annotations | Annotations to be added to the node Deployment | {} |
+| node.podAnnotations | Annotations to be added to the node pod(s) | {} |
+| node.podAnnotationConfigChecksum | Add Config Checksum to pod Annotations.<br>This will trigger Rolling Updates on configuration changes. | false |
+| node.podSecurityContext | Security Context policies for the node pod(s) | {} |
+| node.securityContext | Security Context policies for the node container | {} |
+| node.debug | Set to true for verbose logging. Set to false for normal verbosity | false |
+| node.jsonLogging | Set to true to enable json logging. Set to false for normal logging | false |
+| node.command | Overriding this is typically not necessary, unless you wish to run the node in a different way | ["/bin/stacks-node"] |
+| node.args | Overriding this is typically not necessary, unless you wish to run the node in a different way | ["start", "--config", "/src/stacks-node/config.toml"] |
```

Comment:
Could this be modified so we default to `["start", "--config", "$configFile"]` ? 
I think in the args option, the only thing we'd want to support changing is the config file location

Reply:
We _could_ allow the location of the config file to be modified, but I don't see the reason for doing that. In the project's current state it'll always need a toml config file, and the volumeMount for the config file is hardcoded to the same location:
https://github.com/blockstack/stacks-blockchain/pull/2024/files#diff-405c70b73ffe7336e4a8cd2e3d655cf1209d535e9f09ffcfc1d37753eca894ecR69-R71

I could calculate the location in `_helpers.tpl` so it's defined in one place, but I don't think we need to add this as a changeable field in `values.yaml` unless there's a reason to do so.

Reply:
resolving this comment - for now it's fine, we should look again once we get closer to mainnet if it still makes sense to keep this as-is (along with any other potential changes for mainnet)